src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/button-has-type */
+import React, { Component } from 'react';
+import './Calculate.css';
+
+const createDigit = () => {
+  const digits = [];
+  for (let i = 9; i >= 1; i -= 1) {
+    digits.push(<button key={i}>{i}</button>);
+  }
+  return digits;
+};
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Calculator extends Component {
+  render() {
+    return (
+      <div className="calculate">
+        <div className="display"><span>0</span></div>
+        <div className="calc-digits">
+          <div className="calc-left">
+            <div className="operations-top">
+              <button>AC</button>
+              <button>+/-</button>
+              <button>%</button>
+            </div>
+            <div className="digits">
+              {createDigit()}
+              <button>0</button>
+              <button>.</button>
+            </div>
```

Comment:
- [ ] Good job working on this component 👏🏿  please add the border between each button so your design looks exactly like the original design 👍🏿 

Original Design          |  Your design     
:-------------------------:|:-------------------------:
<img width="558" alt="calculator" src="https://user-images.githubusercontent.com/69638013/170066184-a2d353dd-0d63-4500-a5a6-acdb8e694193.png">  | ![Screenshot_213](https://user-images.githubusercontent.com/69638013/170066317-cbb38b36-9dc0-478e-a998-e6323164f0e2.png)